### PR TITLE
[iOS] Frames' content update

### LINKED ISSUE #19127: Triggers are not working on Frame control in .net maui
### Description

Triggers are not working on Frame control in .net maui

### Steps to Reproduce

1. Run the sample app from the link below
2. Click the button "Toggle Camera View". True / False represents the current toggle state.
3. Content on the "Frame" does not update but content on "Border" gets updated.

FrameTriggerBug

### Link to public reproduction project repository

https://github.com/nikamvd/TriggersMauiApp

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Yes, this used to work in Xamarin.Forms

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 16.5.1

### Did you find any workaround?

Works if I replace Frame with Border

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by kubaflo**
### Description of Change

[iOS] fixed updating frame's content

### Issues Fixed

Fixes https://github.com/dotnet/maui/issues/19127

|Before|After|
|---|---|
|<video src="https://github.com/dotnet/maui/assets/42434498/2ae7eb2b-c632-480a-8c1e-75e4adfcc7e0"/>|<video src="https://github.com/dotnet/maui/assets/42434498/98e81879-3c61-4cd6-b070-0b72a1a6189c"/>|

### CODE REVIEW

File: src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs [hunk 1]
```diff
@@ -69,6 +69,21 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				SetupLayer();
 			else if (e.PropertyName == Controls.Frame.HasShadowProperty.PropertyName)
 				UpdateShadow();
+			else if (e.PropertyName == Microsoft.Maui.Controls.Frame.ContentProperty.PropertyName)
```
Comment (jsuarezruiz) [inline src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs:72]: Not related with your changes, but ideally, this Renderer should use the Mapper on iOS already happen on Android: https://github.com/dotnet/maui/blob/d749dc779531dcf491746fd2f190382db4f18e0d/src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs#L28

File: src/Controls/tests/UITests/Tests/Issues/Issue19127.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		}
+
+		[Test]
+		public void ContentOfFrameShouldChange()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
+
+			_ = App.WaitForElement("button");
+
+			App.Click("button");
```
Comment (jsuarezruiz) [inline src/Controls/tests/UITests/Tests/Issues/Issue19127.cs:22]: Could you also verify the text of the Frame Content before tap the Button?. Should be "Camera is Disabled".

File: src/Controls/tests/UITests/Tests/Issues/Issue19127.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+		public override string Issue => "[iOS] Triggers are not working on Frame control";
+
+		public Issue19127(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public void ContentOfFrameShouldChange()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
```
Comment (jsuarezruiz) [inline src/Controls/tests/UITests/Tests/Issues/Issue19127.cs:18]: I know this is an iOS issue, but I think this test could have sense to validate the same behavior on other platforms. Could we pass the test also on Android and Windows?

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

File: src/Controls/tests/UITests/Tests/Issues/Issue19127.cs (hunk 3 above)
Comment (kubaflo) [reply src/Controls/tests/UITests/Tests/Issues/Issue19127.cs:18]: Of course

File: src/Controls/tests/UITests/Tests/Issues/Issue19127.cs (hunk 2 above)
Comment (kubaflo) [reply src/Controls/tests/UITests/Tests/Issues/Issue19127.cs:22]: That's a good idea

File: src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs (hunk 1 above)
Comment (kubaflo) [reply src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs:72]: I actually wanted to do it in the first place, but I wasn't sure if I should have 😅

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**